File: C++/Chains/MdsOmElement.cpp

#include "MdsOmChains.h"

namespace MdsOmNs {

MdsOmElement::MdsOmElement()
{
}

MdsOmElement::~MdsOmElement()
{
}

int MdsOmElement::init(const char *elementName)
{
	 name = elementName;
	 dirty = true;
	 return 0;
}

}

File: Java/MdsOmHelperClasses/src/com/local/mds/mercury/helper/chain/MdsOmElement.java

package com.jpmorgan.mds.mercury.helper.chain;

class MdsOmElement {
	String name = "";
	boolean dirty = false;

	MdsOmElement(String name)
	{
		this.name = name;
	}
	
	String getName() { return name; }
	
	void setName(String name) { this.name = name; }
	
	boolean isDirty() { return dirty; }
	
	void setDirty(boolean dirty) { this.dirty = dirty; }

}
